cfdp-core/src/daemon.rs

```diff
@@ -572,7 +572,7 @@ impl<T: FileStore + Send + Sync + 'static> Daemon<T> {
 
         for (vec, mut transport) in transport_map.into_iter() {
             let (pdu_send, pdu_receive) = unbounded();
-            let (remote_send, remote_receive) = unbounded();
+            let (remote_send, remote_receive) = bounded(1);
```

Comment:
bounded(1) lets one PDU buffer in the channel right? One PDU isn't much really, but I remember we did talk about wanting things like the EoF to send immediately. Just want to make sure we don't want bounded(0) here, which would block until the transport could receive the PDU from the transaction.

Reply:
bounded(0) does not work currently because the transport is not blocked on reading from the channel (it just tries from time to time to read with try_recv() ) so the select will never return that the channel is writable.

But I think it's ok like that and we might even increase 1 to N to allow the transport to send PDUs in bursts which can then be scheduled by the Linux OS to go at the right bitrate.
Currently the sleep(500 micros) in the transport causes quite some overhead and the PDUs are sent one each 700 micros with quite some jitter. Most of the time people want to configure the maximum bitrate not the PDU rate (because not all PDUs are the same size, the two are not exactly equivalent).
I will make an issue for that after some more experimentation.

Reply:
That makes sense. Thanks for the explanation. The sleeps that are there by default were purely random honestly. We should look into better options I think.